src/Select.tsx

```diff
@@ -158,7 +160,11 @@ const customStyles = () => {
       ...provided,
       color: selectTheme.fg,
       backgroundColor: selectTheme.bg,
-      border: selectTheme.border ? `1px solid ${selectTheme.border}` : 'none',
+      border: invalid
```

Comment:
Shall we also make use of [:invalid](https://developer.mozilla.org/en-US/docs/Web/CSS/:invalid) pseudo-class?
We could make use of browser validation (email, pattern etc.) and have similar styling.

Reply:
The main problem with the `:invalid` pseudo-class is it applies before elements have experienced any user interaction. Bit of a design flaw honestly, because then an empty form is bombed with tons of red.

Reading through all the options on https://stackoverflow.com/questions/7920742/delay-html5-invalid-pseudo-class-until-the-first-event it appears one *possible* solution for modernish browsers is:

```
:not(:placeholder-shown):invalid
```

But... that only works if you set a placeholder, and doesn't handle the `required` case.

My personal attitude would be: screw `:invalid` pseudo-class because it's useless, and wait for [`:user-invalid`](https://drafts.csswg.org/selectors-4/#user-pseudos) when we get CSS4.